src/Components/DraggableResizer/DraggableResizer.tsx

```diff
@@ -162,16 +193,21 @@ export const DraggableResizerRoot: FunctionComponent<DraggableResizerProps> = ({
         if (child.type !== DraggableResizerPanel) {
           return null
         }
-
         const isLastPanel = i === panelsCount - 1
         const dragging = i === dragIndex
 
+        const isCollapsible0 = child.props.isCollapsible
```

Comment:
Same comment as Bucky, I can't understand the intent of the 0 / 1 in the variable name, we should use something that's easier to follow and is english.